CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Welcome to the i18n Translation Generator contributing guide
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on docs.github.com ✨.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the README file. Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests)
+
+## Getting started
+
+There are a lot of things that you could do to collaborate on open sources projects!. Will leave you some ideas about what could you do to help us to improve i18n translation generator
+
+### Issues
+
+#### Create a new issue
+
+If you spot any problem while using the project, see any error on the doc, or just wants to suggest changes to improve the project you can [create a new issue](https://github.com/victor-heliomar/i18n-translation-generator/issues/new/choose). Please make sure of check first that the issue doesn't exists already exists on the [issues list](https://github.com/victor-heliomar/i18n-translation-generator/issues)
+
+#### Solve an issue
+
+Scan through our existing issues to find one that interests you. If you find an issue to work on, you can leave a comment on the issue to leave us know that you're working on it and when you had developed your solution you can open a PR with a fix.
+
+#### Help to solve the community doubts
+
+If there is any doubt on any issue and you think you can help to solve it, then feel free to help to clear the doubts throught the comments box of the issue selected, always following our [CODE OF CONDUCTS](https://github.com/victor-heliomar/i18n-translation-generator/blob/dev/CODE_OF_CONDUCT.md)
```

Comment:
Maybe you can change the URL a little 
```suggestion
If there is any doubt on any issue and you think you can help to solve it, then feel free to help to clear the doubts throught the comments box of the issue selected, always following our [CODE OF CONDUCTS](https://github.com/victor-heliomar/i18n-translation-generator/CODE_OF_CONDUCT.md)
```

Reply:
Proposed link seems like is not valid
![image](https://github.com/victor-heliomar/i18n-translation-generator/assets/66505715/a3a4a6d2-ae5c-40e2-8204-c2a661b936b4)

Reply:
But fixed the typo and added final end on each paragraph.- Thanks!